packages/enzyme-utilities/src/wrappers.ts

```diff
@@ -0,0 +1,25 @@
+import {ReactWrapper} from 'enzyme';
+
+const mountedWrappers: ReactWrapper<any, any>[] = [];
```

Comment:
IMO, we should be using a factory that bends this stuff and keeps the mounted wrappers in the closure, I feel this is a bad pattern for library code and it would make your tests more meaningful